lnrpc-ideal/routerrpc/router.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+import "rpc.proto";
+
+package routerrpc;
+
+option go_package = "github.com/lightningnetwork/lnd/lnrpc/routerrpc";
+
+message TrackPaymentRequest {
+    /// The hash of the payment to look up.
+    bytes payment_hash = 1;
+}
+
+service Router {
+    /**
+    TrackPayment returns an update stream for the payment identified by the
+    payment hash.
+    */
+    rpc TrackPayment (TrackPaymentRequest) returns (stream lnrpc.Payment);
```

Comment:
We don't have a `LookupPayment` call, but analogue to `SubscribeSingleInvoice` this call can be used for that as well.